.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,15 @@
 # Title
 
-<!-- Thanks for sending a pull request! Make sure to follow the contributing guidelines. -->
+<!-- Thanks for sending a pull request! Make sure to follow the contributing guidelines: https://github.com/FreeTubeApp/FreeTube/blob/development/CONTRIBUTING.md -->
 <!-- Important note, we may remove your pull request if you do not use this provided PR template correctly. -->
 
 ## Pull Request Type
 <!-- Please select what type of pull request this is: [x] -->
+<!-- Please remove what is not applicable! -->
```

Comment:
I read this and now I am confused

Reply:
So all the options in the pull request type headers are keywords that triggers labels. If something is a bugfix they should select bugfix and delete the other keywords because then only the bugfix label gets applied. If they dont delete the other keywords or forget to delete a few. All keywords will trigger labels being applied.

Hopefully i didnt explain it to complicated

Reply:
If you want people to remove what's not applicable, then no point asking them to select `what type of pull request this is`
Confusing instructions = bigger chance of be carried out accurately

Reply:
Or is it possible to set config like
```yaml
autolabeler:
  - label: 'PR: enhancement'
    body:
      - '- [x] Feature Implementation'
  - label: 'PR: bug'
    body:
      - '- [x] Bugfix'
```

Anyway it seems very weird to have both sentences here

Reply:
SMH🤦‍♂️ why didnt i think about that. Tested in https://github.com/FreeTubeApp/Test-Repo/pull/119 and it worked. What would i do without u?

Reply:
https://github.com/FreeTubeApp/Test-Repo/pull/119 doesn't seem a proper test
Created https://github.com/FreeTubeApp/Test-Repo/pull/145
The issue is we can't enforce people to only select one for PR type (let's hope that doesn't happen often

Reply:
@PikachuEXE should i maybe clarify with a comment to choose one?

Reply:
Clearer is better

Also how to make it like you can only choose one Pokemon 😆 
![image](https://user-images.githubusercontent.com/1018543/200977260-df267474-69e4-4342-9611-2058bb87a333.png)